store-on-wpcom/class-wc-calypso-bridge.php

```diff
@@ -32,7 +32,10 @@ class WC_Calypso_Bridge {
 	 * Constructor.
 	 */
 	public function __construct() {
-		add_action( 'woocommerce_init', array( $this, 'init' ), 20 );
```

Comment:
Can we remove `private function init() {` from this file with this change too?

Reply:
🤦‍♂️ Thanks for catching that. Removed.